src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4153,7 +4153,8 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Tag'
-
+        lockingJobId:
```

Comment:
Seeking reviewer feedback:

I was on the fence as to whether we'd be better served to future-proof this response by including any shared locks as well.  A compromise might be to instead expose this exclusive lock in an encapsulating structure instead, like:

```
locks: {
  exclusive: <an-exclusive-lock>
}
```

And if down the road, we wanted to expose shared locks, we could add them to the structure without breaking consumption.

Thoughts?

Reply:
(Ah! And I think I've just convinced myself that `lockingJobId` is not a future-looking name for this field. We may want to allow for locks that aren't job IDs.)

Reply:
Thanks!  I've decided to change this to the encapsulating structure above for future-proofing.